test/util.js

```diff
@@ -169,6 +169,18 @@ return {
 	/**
 	 * Etc
 	 */
+	assertRuntimeBind: function( assert, formatter, runtimeKey, generatorString, runtimeArgsFn ) {
+		assert.ok( formatter );
+		assert.equal( typeof formatter, "function" );
+		assert.ok( "runtimeKey" in formatter );
+		assert.equal( formatter.runtimeKey, runtimeKey );
+		assert.ok( "generatorString" in formatter );
+		assert.equal( typeof formatter.generatorString, "function" );
+		assert.equal( formatter.generatorString(), generatorString );
+		assert.ok( "runtimeArgs" in formatter );
+		runtimeArgsFn( formatter.runtimeArgs );
```

Comment:
Actually, this allows a custom tests to be performed by each calling code. For example: https://github.com/jquery/globalize/pull/436/files#diff-9ee4a6f38e08bc680f4668d25dedaff8R30, https://github.com/jquery/globalize/pull/436/files#diff-3b257e2c8b293a57b0336b413514f034R110. Please, just let me know if I misinterpreted your comment.